assets/theme-css/content.css

```diff
@@ -1,15 +1,32 @@
+/* Default: single column layout for mobile */
+.content-container {
+  max-width: 70em;
+  margin: 0 30px;
+  flex-direction: column;
+}
+
+/* Full screen two-column layout for desktop etc. */
+@media only screen and (min-width: 75em) {
```

Comment:
If the flex basis for this one is 70em, and for the shortcuts 15em, then I suppose we want to activate this at 85em?